sllurp/verb/access.py

```diff
@@ -5,9 +5,8 @@
 import sys
```

Comment:
I'm fine with the changes in this file, although they're not really related to the addition of a `capabilities` command. Looks like you used `black` or similar?

Reply:
It seems this file has been modified by black before I disabled it. I would like to remove this line
``` python
from sllurp.llrp_proto import Modulation_DefaultTari
```

 which leads to an error with cli because Modulation_DefaultTari is no longer existing in llrp_proto.py

I will try to remove black formatting on this file.

Reply:
Done in commit d49d94e2e95f0c8d6659ec5c0537fcb1995c8e35